pkg/machines/helpers.es6

```diff
@@ -87,14 +90,24 @@ export function logDebug(msg) {
     }
 }
 
-export function rephraseUI(key, original) {
-    const transform = {
-        'autostart': {
-            'disable': 'disabled',
-            'enable': 'enabled'
-        }
-    };
+const transform = {
+    'autostart': {
+        'disable': _('disabled'),
```

Comment:
We use the gettext C parser, so translatable strings need to have double quotes.